types/react-map-gl/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-map-gl 3.3
+// Type definitions for react-map-gl 4.0
```

Comment:
If you think the type definitions for version 3 will need to be updated in the future, please follow the guide [here](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version). Otherwise, I will go ahead and merge this.

Reply:
I don't think the version 3 definitions will need any updates. Hence I would be glad if you could merge. Thanks!